api-testing/multicloud-api/README.md

```diff
@@ -0,0 +1,27 @@
+# API Testing Automation Framework
+
+**Introduction**
+
+This framework has developed in [JUNIT](https://junit.org/) using [Gradle](https://gradle.org/) Build tool.
+Currently added for multicloud project of soda foundation. Later it could be reused for other projects.
+Junit framework details [Link](https://www.tutorialspoint.com/junit/index.htm)
+
+**Why this?**
+
+Junit is widely used testing framework along with Java Programming Language. You can use this automation framework for both unit testing and UI testing.
+
+**How to install and use?**
+
+ Download Intellij idea community edition tool and install in your pc.
+ [link](https://www.jetbrains.com/idea/download/#section=windows)
+ 
+ Click on File > Open or  You Can create new Project click on File > Project
+ 
+ Get a project from version control [Link](https://www.jetbrains.com/help/idea/import-project-or-module-wizard.html)
+ 
+**How to add tests?**
+
+ In Project structure `src\test\java` write your own test case and 
```

Comment:
Seems no src\etst\jave exist, could you plz add a little more explanation, or just give a simple example?

Reply:
Please refer this [link ](https://docs.gradle.org/current/userguide/organizing_gradle_projects.html)

Reply:
That's good, could you please add this link to README?

Reply:
I have added Please review.